src/main/java/uk/gov/justice/hmpps/prison/api/model/VisitSummary.kt

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.hmpps.prison.api.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.swagger.annotations.ApiModel
+import io.swagger.annotations.ApiModelProperty
+import java.time.LocalDateTime
+
+/**
+ * Visit summary
+ */
+@ApiModel(description = "Visit summary")
+data class VisitSummary(
+  @ApiModelProperty(value = "Date and time at which next scheduled (i.e. not cancelled) event starts (if any)")
+  @JsonProperty("startTime")
+  val startTime: LocalDateTime?,
+
```

Comment:
very minor but would startDateTime be better. Confusingly Visits seem to be modelled with a Date and Start time and End time. So this was a little confusing until I read the comment